pyproject.toml

```diff
@@ -26,6 +26,10 @@ dependencies = [
 test = [
   "pytest",
   "wait_for",
+  "pre-commit",
```

Comment:
those are not test dependencies, i prefer to confine them to pre-commit and/or add a typing extra